subsys/net/ip/ipv6.c

```diff
@@ -159,6 +159,15 @@ static inline enum net_verdict process_icmpv6_pkt(struct net_pkt *pkt,
 		return NET_DROP;
 	}
 
+#define MIN_ICMPV6_LEN (1 + 1 + 2 + 2 + 2) /* type, code, checksum,
```

Comment:
Having such macro here is not nice, could you put it in icmpv6.h, have a name prefixed NET_ICMPV6_ and reuse also NET_ICMPV6H_LEN for the hdr part (type, code, chksum).